src/lib/dvf/bindApi.js

```diff
@@ -77,14 +78,13 @@ module.exports = () => {
     getNetwork: compose(require('../../api/eth/getNetwork'))
   }
 
-  //dvf utility functions
+  // dvf utility functions
   dvf.util = {
     generateRandomNonce: require('./generateRandomNonce')
   }
 
   // dvf.sign functions
   dvf.sign = compose(require('../../api/sign/sign'))
-  //dvf.sign.cancelOrder = compose(require('../../api/sign/cancelOrder'))
```

Comment:
Is this because in future we will only sign cancel requests with nonceSignature, instead of signing the orderId ?

Reply:
Yes that's right. this method is now legacy and has been commented out in bindApi for some time now.